stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ApplicationDescriptor.java

```diff
@@ -61,8 +63,38 @@ protected void onGetChildren(Application element, Accumulator<Object> children)
         children.store(activity);
       }
     }
+    storeWindowIfNeeded(element,children,activities);
   }
+  private void storeWindowIfNeeded(Application application, Accumulator<Object> children, List<WeakReference<Activity>> activities){
+    List<View>windows = WindowFetcher.getWindowViews(application);
+    if (windows == null){
+      return;
+    }
+    for (View view : windows) {
+      if (belongToActivity(view, activities)) {
+        continue;
+      }
+      children.store(view);
+    }
+  }
+
+
 
+  static boolean belongToActivity(View view, List<WeakReference<Activity>> references) {
+    if (references == null) {
```

Comment:
the null checking through these edits seems excessive.  can you update the code to use `@Nullable` in all the places you genuinely expect nullness and drop the checks for the others / convert them into `Util.throwIfNull` calls.